models/TradingAccount.py

```diff
@@ -289,345 +294,45 @@ def saveTrackerCSV(self, market='', save_file='tracker.csv'):
+                            df_buy['type'],
                             df_buy['size'],
                             df_buy['value'],
-                            df_buy['fees'], 
+                            df_buy['fees'],
                             df_buy['price'],
                             df_sell['created_at'],
-                            df_sell['type'], 
-                            df_sell['size'], 
+                            df_sell['type'],
+                            df_sell['size'],
                             df_sell['value'],
-                            df_sell['fees'], 
-                            df_sell['price']                    
-                        ]], columns=[ 'status', 'market', 
-                            'buy_at', 'buy_type', 'buy_size', 'buy_value', 'buy_fees', 'buy_price',
-                            'sell_at', 'sell_type', 'sell_size', 'sell_value', 'sell_fees', 'sell_price' 
-                        ])
+                            df_sell['fees'],
+                            df_sell['price']
+                        ]], columns=['status', 'market',
+                                     'buy_at', 'buy_type', 'buy_size', 'buy_value', 'buy_fees', 'buy_price',
+                                     'sell_at', 'sell_type', 'sell_size', 'sell_value', 'sell_fees', 'sell_price'
+                                     ])
                     df_tracker = df_tracker.append(df_pair, ignore_index=True)
                     pair = 0
-                
+
                 last_action = row['action']
 
-        if list(df_tracker.keys()) != [ 'status', 'market', 
-                            'buy_at', 'buy_type', 'buy_size', 'buy_value', 'buy_fees', 'buy_price',
-                            'sell_at', 'sell_type', 'sell_size', 'sell_value', 'sell_fees', 'sell_price' ]:
+        if list(df_tracker.keys()) != ['status', 'market',
+                                       'buy_at', 'buy_type', 'buy_size', 'buy_value', 'buy_fees', 'buy_price',
+                                       'sell_at', 'sell_type', 'sell_size', 'sell_value', 'sell_fees', 'sell_price']:
             # no data, return early
             return False
 
-        df_tracker['profit'] = np.subtract(np.subtract(df_tracker['sell_value'], df_tracker['buy_value']), np.add(df_tracker['buy_fees'], df_tracker['sell_fees']))
+        df_tracker['profit'] = np.subtract(np.subtract(df_tracker['sell_value'], df_tracker['buy_value']),
+                                           np.add(df_tracker['buy_fees'], df_tracker['sell_fees']))
         df_tracker['margin'] = np.multiply(np.true_divide(df_tracker['profit'], df_tracker['buy_value']), 100)
         df_sincebot = df_tracker[df_tracker['buy_at'] > '2021-02-1']
 
         try:
             df_sincebot.to_csv(save_file, index=False)
         except OSError:
-            raise SystemExit('Unable to save: ', save_file) 
-
-    def buy(self, cryptoMarket, fiatMarket, fiatAmount=0, manualPrice=0.00000000):
```

Comment:
The buy method used on pycryptobot is `PyCryptoBot::marketBuy`

Reply:
@whittlem Do you remember why those duplicate functions?
I was thinking to remove them when I did the big refactoring, but it was already too much...